lib/chef/provider/git.rb

```diff
@@ -105,7 +105,7 @@ def action_sync
       end
 
       def git_minor_version
-        @git_minor_version ||= Gem::Version.new(shell_out!("git --version", run_options).stdout.split.last)
+        @git_minor_version ||= Gem::Version.new((run_git_command ["--version"]).stdout.split.last)
```

Comment:
Should we go one step further here and write it like this?

`@git_minor_version ||= Gem::Version.new git("--version").stdout.split.last`